pkg/gameservers/controller.go

```diff
@@ -109,6 +110,12 @@ func NewController(
 	c.recorder = eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: "gameserver-controller"})
 
 	c.workerqueue = workerqueue.NewWorkerQueue(c.syncGameServer, c.logger, stable.GroupName+".GameServerController")
+	health.AddLivenessCheck("game-server-worker-queue", healthcheck.Check(func() error {
+		if !c.workerqueue.Healthy() {
```

Comment:
Thought - would it make sense to have c.workerqueue.Healthy() return an error when it's not healthy - rather than a bool?

Then we could just do: `health.AddLivenessCheck("game-server-worker-queue", c.workerqueue.Healthy)`

Not wedded to it, but thought it might be worth discussion?

Reply:
That's a better idea. I'll make the change.